src/utils.rs

```diff
@@ -9,16 +9,15 @@
 
 use std::borrow::Cow;
 use std::io::Cursor;
+use std::io::Read;
 use std::io::Seek;
+use std::io::SeekFrom;
 use std::io::Write;
 use std::marker::PhantomData;
 use std::os::fd::OwnedFd;
 
 use anyhow::anyhow;
 use anyhow::Context;
-use byteorder::ReadBytesExt;
```

Comment:
What is motivation for below changes? 
- Remove bytes and bitreader dependency
- enumn

Reply:
The motivation is just to remove the dependency on the bytes, bitreader, and enumn crates. Every crate that we depend on, including transitive dependencies, must go through Chrome's third party library approval process. Plus, it's a security liability. I'd like to eliminate as many dependencies as possible before integration with Chrome to streamline the process.